src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+/**
+ * Represents a <code>Todo</code> type of task
+ */
+public class Todo extends Task {
+
+    /**
+     * Constructs an instance of <code>Todo</code>
+     * @param body Body message
+     */
+    Todo (String body) {
+        super(body, false);
+    }
+
+    /**
+     * Constructs an instance of <code>Todo</code>
+     * @param body Body message
+     * @param done Status of task
+     */
+    Todo(String body, boolean done) {
+        super(body, done);
+    }
+
+    /**
+     * Returns <code>Todo</code> with done set to true
+     * @return Done <code>Todo</code>
+     */
+    @Override
+    Task setDone() {
+        return new Todo(this.getBody(), true);
```

Comment:
It would be easier to use the parent class's `setDone()` method (after changing it as per the previous comment) so you would not need to override this method in all the different task types.